docs/content/dependencies/metadata-storage.md

```diff
@@ -57,20 +62,20 @@ metadata storage you plan to use. See below.
 3. Configure your Druid metadata storage extension:
 
   Add the following parameters to your Druid configuration, replacing `<host>`
-  with the hostname of the database.
+  with the location (host name and port) of the database.
 
   ```properties
   druid.extensions.loadList=["mysql-metadata-storage"]
   druid.metadata.storage.type=mysql
-  druid.metadata.storage.connector.connectURI=jdbc:mysql://<host>/druid_test
+  druid.metadata.storage.connector.connectURI=jdbc:mysql://<host>/druid
   druid.metadata.storage.connector.user=druid
   druid.metadata.storage.connector.password=diurd
   ```
 
-  Note: metadata storage extension is not packaged within the main Druid tarball, it is
+  Note: the metadata storage extension is not packaged within the main Druid tarball; it is
```

Comment:
I did not find a metadata storage tarball at the link.  That link is used for this purpose on the first-look tutorials page, and maybe elsewhere.